src/list.cpp

```diff
@@ -0,0 +1,89 @@
+/** @file list.cpp
+ * @brief Unit tests for Assignment Classes: practice on classes and user
+ *   defined types.
+ *
+ * @author Mathis Rhoades
+ * @note   class: COSC 2336, Summer 2024
+ * @note   ide  : VSCode Server 3.9.3, Gnu Development Tools
+ * @note   assg : Assignment STL
+ * @date   December 5, 2024
+ *
+ * Assignment Standard Template Library STL.
+ */
+#include "list.hpp"
+#include <iostream>
+#include <list>
+#include <queue>
+#include <sstream>
+#include <string>
+using namespace std;
+
+/**
+ * @brief Queue<int> to string
+ *
+ * Accessor method to construct and return a string representation
+ * of the current values and status of this Queue instance.
+ *
+ * @returns string Returns the string constructed with the information
+ *   about this Queue.
+ */
+string str(queue<int> queue)
+{
+  // create string of text
+  ostringstream out;
+
+  // stream queue information into the output stream
+  out << "<queue> size: " << queue.size() << " front:[ ";
+
+  // stream the current values of the queue to the output stream
+  // start with values in format until the last value
+  while(!queue.empty() and queue.size() > 1)
+{
```

Comment:
This code has not been run through the class style checker/formatter.  Indentation issues here.